recipes/qsv/bld.bat

```diff
@@ -0,0 +1,14 @@
+:: check licenses
+cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+
+:: build
+cargo install --root "%LIBRARY_PREFIX%" --path . --all-features || goto :error
```

Comment:
@peterjc , as I understand it, bld.bat is used to build for Windows.

I'd change this line to:
`cargo install --root "$PREFIX" --path . --locked --features feature_capable,apply,generate,foreach,fetch,geocode`

Reply:
the `--all-features` option here explains why hashbrown is being built with nightly, as we do have a `nightly` feature in qsv Cargo.toml.

Reply:
Ah yes, I overlooked the Windows options were in a separate file.